### PR TITLE
Feat/heroku

### PR DESCRIPTION
**Pull request by edmdc**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- No Heroku deployment

## New behavior

- Deployed to Heroku. Should be automatic from main branch, but will tweak if not.

- Should be viewable ![here] (https://cash-moneth.herokuapp.com)

## Other information

Appearance change a little bit 

image

### CODE REVIEW

**Review by jpc20 — COMMENTED** [review]
Nice job getting this deployed! Looks like there are a few checks failing, do you know what those are?

File: package.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "postinstall": "if test \"$NODE_ENV\" = \"production\" ; then npm run build ; fi",
+    "start": "node server.js",
```
Comment (edmdc) [inline package.json:9]: These commands should allow for continuous deployment to heroku

File: server.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const server = require("express")();
+const path = require("path");
+const serveStatic = require("serve-static");
+
+server.use(serveStatic(path.join(`${__dirname}/dist`)));
+
+const port = process.env.PORT || 5000;
+server.listen(port);
+
+console.log(`Begin your campaign on http://localhost:${port}`);
```
Comment (edmdc) [inline server.js:10]: Quick little server to serve our dist folder (hopefully created by the post-install script on package.json) to heroku

Comment (attom2) [reply server.js:10]: Very neat, like we are bypassing the need to build all of the complex files

File: package.json (hunk 1 above)
Comment (attom2) [reply package.json:9]: Wow that first one is a beast

**Review by attom2 — APPROVED** [review]
Good work!

**Pull request merged**